data/language/pl-PL.yml

```diff
@@ -2209,21 +2155,21 @@ strings:
   2110: Należy wybrać typ klimatu
   2111: Należy wybrać typ generatora mapy
   2112: Należy wybrać typ regionu
-  2113: mph
-  2114: kmh⁻¹
-  2115: 'hour:'
-  2116: 'hours:'
-  2117: mins
+  2113: mil/h
+  2114: km/h
+  2115: 'godzina:'
+  2116: 'godzin:'
+  2117: minut
   2118: 'min:'
-  2119: secs
-  2120: ' units'
-  2121: ft
+  2119: sek.
+  2120: ' jednostek'
+  2121: st
   2122: m
-  2123: hp
+  2123: KM
   2124: kW
   2125: "{COLOUR WINDOW_2}Język:"
-
-  2127: "{COLOUR WINDOW_2}Tryb pełnego ekranu:"
+  2126: Wyłącz awarie pojazdów
+  2127: "{COLOUR WINDOW_2}Tryb wyświetlania:"
   2128: Okno
-  2129: Pełny ekran
-  2130: Pełny ekran w oknie
+  2129: Pełen Ekran
+  2130: Bez Krawędzi
```

Comment:
Is this a better translation than `Pełny ekran w oknie`? It seems to be missing the 'window' part.

Reply:
Some of the options in Loco were capitalised some of them not so I decided to make them equal in selection boxes...